.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+# Prerequisites
+
+Please make sure you've checked the following to-do's.
+
+- [ ] My ad-block or script blocker is disabled (if you have any).
+- [ ] My browser is IE or EDGE.
+- [ ] I checked that there are no browser extensions that may affect the issue on my browser.
+- [ ] This is not a duplicated issue, i checked current issues before creating this issue.
+
+# Expected Behavior
+
+Please describe the behavior you are expecting
+
+# Current Behavior
+
+What is the current behavior?
+
+# Failure Information (for bugs)
+
+Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.
```

Comment:
We can explain something different here since this is a bug report template already.